src/librustc_parse/parser/mod.rs

```diff
@@ -662,6 +662,21 @@ impl<'a> Parser<'a> {
                                 }
                             }
 
+                            // If this was a missing `@` in a binding pattern
+                            // bail with a suggestion
+                            // https://github.com/rust-lang/rust/issues/72373
+                            if self.prev_token.is_ident() && &self.token.kind == &token::DotDot {
+                                expect_err
+                                    .span_suggestion_short(
```

Comment:
Because of the wording suggestion, at that point it makes sense to change it to be 
```suggestion
                                    .span_suggestion_verbose(
```

so that the content is shown in its own subwindow always.

Reply:
changed in 0fdd31317a9c8d2fbc284546c711bc5d5139d83b